.eslintignore

```diff
@@ -1,5 +1,8 @@
-**/*test.js
-/**/*.d.ts
-/node_modules/
-/dist/**/*.js
+dist/
+reports/
+.husky/
+__mocks__/
+*.json
+*.md
+*.d.ts
```

Comment:
We create this declaration file manually. Hence, it's better to have it.

Reply:
we cannot lint ts files without typescript configuration, lets add this is when we start moving to TS